Block cjr.org interstitial popup

wavygecko opened the pull request:
## What issue is being fixed?

Block interstitial popup on cjr.org (e.g. `https://www.cjr.org/the_media_today/laflamme_canada_media_diversity.php`)

<details>

<summary>Screenshot 1:</summary>

![image](https://user-images.githubusercontent.com/110956608/187594466-c637e32e-8253-4884-9fd9-40c432ae920b.png)

</details>

wavygecko committed:
Block cjr.org interstitial popup

zloyden commented on AnnoyancesFilter/Popups/sections/popups_specific.txt:
```diff
@@ -7,6 +7,8 @@
 !
 ! SECTION: Popups - Regular rules
 !
+cjr.org###interContainer
+cjr.org###interVeil
```
Looks like the ad leftover. Add the rules to the AdGuad Base - general_extensions.txt near the similar rules `#$#` with the pull request comment.
```
! https://github.com/AdguardTeam/AdguardFilters/pull/128485
cjr.org#$#html { overflow: auto !important; }
cjr.org#$##interContainer { display: none !important; }
cjr.org#$##interVeil { display: none !important; }
```

wavygecko committed:
Update cjr.org popup filters
- re: https://github.com/AdguardTeam/AdguardFilters/pull/128485#discussion_r959248548
